db/schema.rb

```diff
@@ -130,6 +130,13 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
   end
+  
+  create_table "staff", force: :cascade do |t|
```

Comment:
Unfortunately you cannot update `schema.rb` directly in order to add/update db tables.  Instead you'll need to:

1. Create a migration file that creates the new table.  Here's [an example](https://github.com/OperationCode/operationcode_backend/blob/master/db/migrate/20170701172700_create_resources.rb).
2. Run the migration task to update the `schema` file.  With docker running, you can call the [make db_migrate](https://github.com/OperationCode/operationcode_backend/blob/master/Makefile#L35) command.